action.yml

```diff
@@ -55,7 +55,7 @@ outputs:
   result:
     description: Merged configuration as JSON or plain text.
 runs:
-  using: node12
+  using: node16
```

Comment:
The node16 environment has significant differences from previous versions.
So you can fix it. Or I will have to do it, but a little later.